src/Augmentation/AugmentationHelpers.tsx

```diff
@@ -41,16 +40,12 @@ function createAugmentations(): void {
 }
 
 function resetFactionAugmentations(): void {
-  for (const name of Object.keys(Factions)) {
-    if (Factions.hasOwnProperty(name)) {
-      Factions[name].augmentations = [];
-    }
-  }
+  for (const faction of Object.values(Factions)) faction.augmentations = [];
 }
 
 function initAugmentations(): void {
   resetFactionAugmentations();
-  clearObject(StaticAugmentations);
+  for (const augName of Object.getOwnPropertyNames(StaticAugmentations)) delete StaticAugmentations[augName];
```

Comment:
Mental note: Good candidate for future JSONMap

Reply:
Yeah I think StaticAugmentations isn't even from the save file, so it can just be a regular map I think.

The getOwnPropertyNames thing here is weird, I was just keeping the functionality the same from the removed helper function. Iterating through keys should be functionally identical in this case though. But yeah we should just redo it as a map soon.